packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row.stories.js

```diff
@@ -102,6 +102,33 @@ export const argTypesData = {
     },
   },
 
+  displayNameSlot: {
+    name: 'displayName',
+    description: 'Slot for display name',
+    control: 'text',
+    table: {
+      category: 'slots',
+      type: {
+        summary: 'VNode',
+      },
+    },
+  },
```

Comment:
This doesn't work in storybook when you make the prop blank and update the slot with text. This only happens in vue 2, seems to work fine in vue 3.

Reply:
Oh, I took the example from Radio and I see it's not working on Vue 2 there either. Will look into it

Reply:
Seems like it's a problem with SB 7 https://github.com/storybookjs/storybook/issues/16750
It does work when clicking "remount component" though

Reply:
interesting.. lets not worry about it then.